code-challenges.test.js

```diff
@@ -36,14 +81,57 @@ const nums2 = [109, 5, 9, 67, 8, 24]
 
 // b) Create the function that makes the test pass.
 
+const minMax = (array) => {
+  let min = array[0], max = array[0]
+
+for (let i = 1; i < array.length; i++) {
+  let value = array[i]
+  min = (value < min) ? value : min
+  max = (value > max) ? value : max
+}
+
+return [min, max]
+}
+// console.log(minMax(nums1));
+// console.log(minMax(nums2));
+
 
 // --------------------3) Create a function that takes in two arrays as arguments and returns one array with no duplicate values. STRETCH: Use the spread operator to pass in a dynamic number of arguments.
 
+//create function called noDupe
+//Inputs two arrays of Numbers
+//combines both arrays and filters out duplicate Numbers
+//outputs new array
+
 // a) Create a test with an expect statement using the variables provided.
 
+describe("noDupe", () => {
+  it("returns the values of two arrays as one with no duplicate numbers"), () => {
+    expect(noDupe([3, 7, 10, 5, 4, 3, 3], [7, 8, 2, 3, 1, 5, 4])).toEqual([3, 7, 10, 5, 4, 8, 2, 1])
+  }
+})
+
+//ReferenceError: describe is not defined. Jest is out to get me I swear
+
 const testArray1 = [3, 7, 10, 5, 4, 3, 3]
 const testArray2 = [7, 8, 2, 3, 1, 5, 4]
 // Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
 
 
 // b) Create the function that makes the test pass.
+//
+function noDupe(array) {
```

Comment:
1. Follow the pseudocode, there should be two arrays considered for the parameter area
2. Review the concat() method. You have to tell what to add.
3. Now find a method that will remove duplicate values from that variable that is storing this concated array. Think of a hof that will return an array that is shorter in length. Then consider what you are evaluating. Then say what you want to return.